pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con-about-document-attributes.adoc

```diff
@@ -0,0 +1,57 @@
+[id="con-about-document-attributes_{context}"]
+= About document attributes
+
+[role="_abstract"]
+You can define the document attributes in {ProductShortname} in an AsciiDoc file. You can then use these attributes in your content. When {ProductShortname} transforms the content to HTML, the attribute definitions replace the attributes. For example,
```

Comment:
I suggest adding one more step in this flow. You define the attributes in an adoc file, you include the attribute file in the pv2/jupiter config file, and then you can use these attributes. 

I suggest we use the same terminology as the asciidoc documentation, such as `:name-of-an-attribute: value of the attribute` https://docs.asciidoctor.org/asciidoc/latest/attributes/attribute-entries/ 
the value replaces/substitutes the attribute name

Reply:
excellent suggestions, updated.